### DISCUSSION TITLE
Stuck at the start

### DISCUSSION THREAD

**Discussion by RoebideBruijn** [theislab/cellrank#720] [Q&A]
Dear all,

I wanted to use cellrank on my 4 timepoints mouse scRNASeq dataset but I don't understand how to get it working. My adata object comes from cellranger output: `adata = sc.read_10x_h5('../data/raw/filtered_feature_bc_matrix.h5'). `

If I start with: https://cellrank.readthedocs.io/en/stable/cellrank_basics.html. 

I'm already stuck at: `scv.pl.proportions(adata).`
`axis 1 is out of bounds for array of dimension 1`

```
scv.pp.moments(adata, n_pcs=None, n_neighbors=None)
WARNING: Could not find spliced / unspliced counts.
KeyError: 'spliced'
```

Can anyone help me out?

**Comment by Marius1311** [comment]
This is an scvelo error, has nothing to do with CellRank, please move to scVelo: https://scvelo.readthedocs.io/

Basically, you're using a plotting function from scVelo which expected spliced/unspliced counts, which you apparently don't have. It doesn't matter, you can continue, just ignore this.

**Comment by AAA-3** [comment]
Hello! I might be able to help - I'm just a user but basically the scVelo function you described is expecting an anndata with spliced and unspliced counts. Cell Ranger only gives spliced counts I believe. You can use Velocyto and run the original Cell Ranger files through Velocyto to get the spliced and unspliced matrix.

**Comment by sylestiel** [comment]
@AAA-3 
you said above, "you can use Velocyto and run the original Cell Ranger files through Velocyto to get the spliced and unspliced matrix."
Can you provide the script to generate the anndata from Cell Ranger bam file? Can it be done outside of CellRanger app?

Thanks!!

**Comment by AAA-3** [comment]
@sylestiel you can check the example from the velocyto website. I believe that Kallisto tools is also considered  an ideal alternative if you want to check that out. The code I used was taken from the Velocyto website and modiefied. I am copying an example below:

`velocyto run10x -m <mm10_repeat_msk.gtf file address> <address to cell ranger output files> <address to reference GTF file used for the cell ranger matrix>`

The mask GTF file was taken from the  link and instructions from the velocyto tute. The reference GTF file can be downloaded from the cell ranger website for which ever version you used. I retrieved mine from the core sequencing team at my institute.

I took the output of this file for my analysis.

**Reply by sylestiel** [reply]
@AAA-3 

Thank you!
